torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1445,12 +1486,14 @@ def _full_post_state_dict_hook(
         if not state_dict:
             return state_dict
 
-        ignored_param_names = set(self._ignored_param_to_param_name.values())
+        ignored_named_params = self._get_ignored_named_parameters()
+        ignored_named_buffers = self._get_ignored_named_buffers()
+        ignored_names = set(n for n, _ in ignored_named_params)
+        ignored_names.update(n for n, _ in ignored_named_buffers)
         for key in state_dict:
-            # Do not need to clone ignored parameters since they are not
-            # sharded
-            clean_param_name = key.replace(FSDP_WRAPPED_MODULE + ".", "").replace(FPW_MODULE + ".", "")
-            if clean_param_name in ignored_param_names:
+            # Do not need to clone ignored parameters and buffers since they
```

Comment:
Not needed in this PR, but we don't actually even need to clone managed FSDP buffers, because they are not sharded as well